javascript/app.js

```diff
@@ -0,0 +1,481 @@
+
+//A. Q + A
+// How do we assign a value to a variable?
+// We use const, let, and if at all not possible, var, to declare the name of the variable and add an equal sign to initialize the variable.
+// const newVariable = "";
+
+// How do we change the value of a variable?
+//Name the variable and assign a new value. 
+//const newVariable = "string";
+
+//What is pseudocoding and why should you do it?
+//Writing out what you are going to do before you code. Because the majority of your work is narrating what needs to happen and how you're going to do it, and like 20% is just writing it, maybe even less. It's more important to understand exactly what you're doing so that you can reference that when you get lost in actually writing the code
+
+//What percentage of time should be spent thinking about how you're going to solve a problem vs actually typing in code to solve it? 
+// 80/20 rule, thinking/coding. Maybe even 90/10. 
+
+
+
+//B. Strings
+let firstVariable;
+firstVariable = "Hello World";
+firstVariable = 2; 
+secondVariable = firstVariable;
+secondVariable = "Any String";
+// 2. 
+
+const yourName = "Ashley";
+console.log("Hello, my name is " + yourName);
+
+//C. Booleans
+const a = 4;
+  const b = 53;
+  const c = 57;
+  const d = 16;
+  const e = 'Kevin';
+
+  console.log(a < b);
+  console.log(c > d);
+  console.log('Name' == 'Name');
+  console.log(true >= false);
+  console.log(false == false == false == false == false == false > true);
```

Comment:
this doesn't have any meaning. 
answer we were looking for is `(false || false || ......... || true)`